resources/shaders/dxr_ambient_occlusion.hlsl

```diff
@@ -18,15 +18,29 @@
 
 #include "rand_util.hlsl"
 #include "dxr_global.hlsl"
+#include "dxr_functions.hlsl"
+#include "dxr_structs.hlsl"
+#include "material_util.hlsl"
 
 RWTexture2D<float4> output : register(u0); // x: AO value
-Texture2D gbuffer_normal : register(t1);
-Texture2D gbuffer_depth : register(t2);
+
+ByteAddressBuffer g_indices : register(t1);
+StructuredBuffer<Vertex> g_vertices : register(t2);
+StructuredBuffer<Material> g_materials : register(t3);
+StructuredBuffer<Offset> g_offsets : register(t4);
+
+Texture2D g_textures[1000] : register(t5);
+Texture2D gbuffer_normal : register(t1005);
+Texture2D gbuffer_depth : register(t1006);
+
+SamplerState s0 : register(s0);
+
+typedef BuiltInTriangleIntersectionAttributes Attributes;
 
 struct AOHitInfo
 {
   float is_hit;
-  float thisvariablesomehowmakeshybridrenderingwork_killme;
+  float padding;
```

Comment:
the old name was better since it is not padding for alignment and only forfallback